Sources/BuildSystemIntegration/BuildSystemManager.swift

```diff
@@ -78,6 +79,18 @@ fileprivate extension BuildTarget {
   }
 }
 
+fileprivate extension DocumentURI {
+  /// If this is a file URI pointing to a symlink, return the realpath of the URI, otherwise return `nil`.
+  var symlinksResolved: DocumentURI? {
```

Comment:
IMO `symlinksResolved` doesn't really communicate "returns `nil` if not a symlink", maybe it should be something more like `symlinkTarget`?

Reply:
Good idea 👍🏽